packages/blockchain-wallet-v4-frontend/src/data/components/requestBtc/sagas.js

```diff
@@ -7,8 +7,9 @@ export default () => {
 
   const firstStepSubmitClicked = function * (action) {
     try {
-      let { accountIdx, addressIdx, message } = action.payload
+      let { accountIdx, addressIdx, receiveAddress, message } = action.payload
       yield put(actions.core.wallet.setHdAddressLabel(accountIdx, addressIdx, message))
+      yield put(actions.core.kvStore.btc.addAddressLabel(receiveAddress, message))
```

Comment:
We do setHdAddressLabel in a few places. Can you create a saga for redux/data/wallet/sagas that takesLatest when setHdAddressLabel is called and do:

```
yield put(actions.core.kvStore.btc.addAddressLabel(receiveAddress, message))
```